src/oci.c

```diff
@@ -216,6 +216,62 @@ cc_oci_get_config_and_state (gchar **config_file,
 	return false;
 }
 
+
+gboolean
+cc_oci_create_cgroups (struct cc_oci_config *config)
```

Comment:
For consistency with the rest of the codebase, it would be good to have a comment header for this new function.